src/future/standard_library/__init__.py

```diff
@@ -62,7 +62,10 @@
 
 import sys
 import logging
-import imp
+try:
+    import importlib
+except ImportError:
```

Comment:
I think that

```suggestion
if PY2:
```

and moving the block after `from future.utils import PY2, PY3` should be enough (i.e. no change needed to `_find_and_load_module()`), since the `RenameImport` class is only instantiated under Python2.